crds/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = 'crds' %}
-{% set version = '7.0.10' %}
+{% set version = '7.1.0' %}
 {% set number = '1' %}
```

Comment:
@jaytmiller Oops, don't forget to reset `number` to zero whenever you change the `version`. Sorry for forgetting to mention that earlier.